src/common/country-base.ts

```diff
@@ -132,5 +132,7 @@ export const african_countries = [
     'zw',
 ]
 
+export const cpa_plan_countries = ['za', 'ec', 'bw']
```

Comment:
Add a comment about this, `cpa_plan_countries` with a bit of explanation

Reply:
👍